src/PerfView/StackViewer/StackWindow.xaml.cs

```diff
@@ -2926,6 +2926,11 @@ private void FinishInit()
             // Make up a trivial call tree (so that the rest of the code works).  
             m_callTree = new CallTree(ScalingPolicy);
 
+            // Finalize the Preset menu (add default commands)
+            var presets = App.ConfigData["Presets"];
+            m_presets = Preset.ParseCollection(presets);
+            FinishPresetMenu();
```

Comment:
Nit.   Please rename FinishPresetMenu to 'ConfigurePresetMenu' and move the two lines  above this call into that method.